meson.build

```diff
@@ -834,6 +834,15 @@ if not libcap.found()
         libcap = cc.find_library('cap')
 endif
 
+
+# Fedora, CentOS: libbpf packaged from GH mirror
+libbpf = dependency('libbpf', required : false)
+
+# Debian: libbpf packaged from kernel sources
+if not libbpf.found()
+        libbpf = dependency('libbpf-dev', required : true)
+endif
```

Comment:
Hmm, I still don't get what libbpf-dev is here. `dependency` takes a pkg-config name as argument
(see https://mesonbuild.com/Reference-manual.html#dependency). Debian's `libbpf-dev` package has `libbpf.pc` file. I don't think anything except the first check is needed.

Reply:
@keszybz 
You are right, checked that in my fork.